### ISSUE TITLE
Please provide IVY version 

### ISSUE THREAD

**Issue by NashIlli** [michaelbromley/ngx-pagination#387]
When angular compiles the library it says: 
"Encourage the library authors to publish an Ivy distribution."

Could be possible to update the library in order to have an Ivy distribution?

Thanks a lot!

**Comment by michaelbromley** [comment]
Yes, this would be good to do. I'm rather limited for time at the moment, but I would happily accept any help on this if anyone wants to offer. Otherwise I'll get around to it when I can.

**Comment by AngeloZwennes** [comment]
This package is now not working anymore as of Angular 16.
